GTG/core/task.py

```diff
@@ -214,10 +213,8 @@ def set_status(self, status, donedate=None, propagation=False, init=False):
                     if self.has_parent():
                         for p_tid in self.get_parents():
                             par = self.req.get_task(p_tid)
-                            if (par.is_loaded() and par.get_status() in
-                                (self.STA_ACTIVE)):
+                            if (par.is_loaded() and par.get_status() in (self.STA_ACTIVE)):
```

Comment:
Sometimes you removed the parenthesis in `if (...)`, but not here. Not sure if PEP8 likes this, but maybe we should be consistent and remove the `(` and `)` here.

Reply:
Agreed, will remove them